### DISCUSSION TITLE
SEB Network Tab Query

### DISCUSSION THREAD

**Discussion by dhandby** [SafeExamBrowser/seb-win-refactoring#124] [Q&A]
Hi Guys,

Just a guidance question in regards to the networks tab.

I want to allow one specific website but deny the rest and they are in the same domain.

Network tab in SEB 3.2.0
***.******.edu/course/view.php?id=87207     Allow
***.******.edu/course/*     DENY all other courses I have access too
***.******.edu/mod/* DENY

It appears as soon as you use a deny action it blocks the domain entirely.

Is there a way around this?

Thanks,

**Comment by dbuechel** [comment]
Yes, that is correct, the "block" action supersedes the "allow" action, see also https://safeexambrowser.org/windows/win_usermanual_en.html#NetworkPaneFilterSection. To solve your issue, you can simply remove the second rule (`...edu/course/*`), as by default, all requests which are not explicitly allowed will be blocked.
